entrypoint/mixedoscluster/mixedoscluster_test.go

```diff
@@ -0,0 +1,48 @@
+package mixedoscluster
+
+import (
+	"fmt"
+
+	"github.com/rancher/distros-test-framework/pkg/assert"
+	"github.com/rancher/distros-test-framework/pkg/testcase"
+
+	. "github.com/onsi/ginkgo/v2"
+)
+
+var _ = Describe("Test: Mixed OS Cluster", func() {
+
+	It("Starts Up with no issues", func() {
+		testcase.TestBuildCluster(GinkgoT())
```

Comment:
Can we avoid creating the cluster if the config contains hardened parameters?

Reply:
In that case we have to be able to read the server configs. We don't have that ability now but can add it in later iterations

Reply:
Updated Jenkinsfile to handle this condition, please review